recipes/python-confluent-kafka/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set pkgname = "python-confluent-kafka" %}
+{% set name = "confluent-kafka" %}
+{% set version = "0.9.4" %}
+{% set sha256hash = "931bcc3512245ae003c86aa860fb62bbfb4036763a3c2908c50150e412bf0aed" %}
+
+package:
+  name: {{ pkgname|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256hash }}
+
+build:
+  number: 0
+  # There is no win build for librdkafka at this time.
+  # Neither python-avro for py36.
```

Comment:
Might want to send a [re-rendering]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ) PR that way.